homeassistant/components/plugwise/config_flow.py

```diff
@@ -175,14 +176,15 @@ async def async_step_user(
 
             try:
                 api = await validate_gw_input(self.hass, user_input)
-            except InvalidSetupError:
-                errors[CONF_BASE] = "invalid_setup"
```

Comment:
Where did this detection go?

Reply:
This error was related to the Anna-connected-to-Adam case. It is now handled in the config_flow, you added some code covering this case some time ago. So removed from the library in this update.

Reply:
> It is now handled in the config_flow, you added some code covering this case some time ago. So removed from the library in this update.

That is not solving the manual flow. The code added to the config flow handle the auto-discovery. The code now removed handled the manual path.

Reply:
Oh, sorry, then I've misunderstood that change.
I'll put it back.

Reply:
Re-added.